### DISCUSSION TITLE
Is it possible to make JSON query work with graph/time series in grafana

### DISCUSSION THREAD

**Discussion by Tomasz-Kluczkowski** [grafana/grafana-infinity-datasource#55] [Q&A]
hi,

Thx for this plugin - it save a lot of hassle for me!

I tried to convert plain json such as below to a `graph` using the `time series` option but it does not work :(.
It looks like the graph is trying to update the y-scale to the values that it sees but no graph is actually plotted. 

It works for a `gauge` very well but that is not good for my needs. I attach my config for the panel below and the graph view which shows that it reads the values but does not plot the graph.

**Grafana: 7.5.1**
**Infinity: 0.6.0**

Am I missing something?

```json
{
  "active_queues": [
      {
        "name": "name_1",
        "messages_unacknowledged": "123",
        "messages": "200"
      }
  ]
}
```

<img width="1031" alt="Screenshot 2021-03-31 at 14 52 21" src="https://user-images.githubusercontent.com/26039401/113155573-12b77d80-9231-11eb-9ff4-4e0ca9cc1f05.png">

<img width="2576" alt="Screenshot 2021-03-31 at 15 02 29" src="https://user-images.githubusercontent.com/26039401/113156706-257e8200-9232-11eb-8186-ed8160a1dee3.png">

**Comment by yesoreyeram** [comment]
It seems your data doesn't have any timestamps associated with it. So you can't plot it over time. 

The time series format attaches a static single dummy time field to your data so that you can use it in grafana panels like gauge etc. It have nothing to do with the graph. 

Hope this makes clear and answers your question.
